packages/graphile-build-pg/src/plugins/PgConnectionArgCondition.js

```diff
@@ -1,67 +1,66 @@
 // @flow
 import type { Plugin } from "graphile-build";
+
+const defaultPgColumnFilter = (_table, _build, _context) => true;
```

Comment:
I think that's meant to be `_attr` rather than `_table`?

Reply:
I wasn’t sure about this one, the argument, at call time, is `table` in this plugin (and in `PgConnectionArgOrderBy.js` too) should I call them `_attr` in `defaultPgColumnFilter` regardless ?

Reply:
Ah! I clearly wasn't reading carefully enough - that explains your concerns calling it `pgColumnFilter`!

We only want to filter the columns in this PR; so every time we call `pgColumnFilter` we should be passing an `attr`. The orderBy columns come from https://github.com/graphile/graphile-build/blob/master/packages/graphile-build-pg/src/plugins/PgOrderAllColumnsPlugin.js

Reply:
ah yes, I didn’t take enough time figuring out how this plugin worked, maybe you can double check the 2 calls I made to pgColumnFilter in PgConnectionArgOrderBy as I’m not confident I did the right thing™ there either